### PR TITLE
Rocket Launch #14: Create a button component

### PR DESCRIPTION
**Pull request by moustafaelsisy**
* Added `prop-types` to package.json
* Added the common Button component
* Included the material-icons stylesheet in index.html
* Changed site's title

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (ToJen) [inline frontend/public/index.html]: are we going with this for now?

Comment (moustafaelsisy) [inline frontend/public/index.html]: You mean the `Material-Icon` font pack, or the way am including it?

Comment (ToJen) [inline frontend/public/index.html]: The font yeah

Comment (moustafaelsisy) [inline frontend/public/index.html]: Material icons is a fairly good font pack; I was planning on keeping it all the way throughout, but if u have an alternative am fine with that too c:

**Pull request merged**